docs/helm-broker/03-03-create-addons-repo.md

```diff
@@ -184,37 +183,37 @@ spec:
   <summary>
   Git SSH
   </summary>
-  
-  The Git SSH protocol requires an SSH key to authorize with your repository. Setting SSH keys differs among hosting providers. 
-  > **NOTE**: See [this](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#generating-a-new-ssh-key) document to read about the GitHub service. 
-  
-  The private key must be base64 encoded.
-  
-  To encode your private key, run this command:
+
+  The Git SSH protocol requires an SSH key to authenticate your repository. Setting SSH keys differs among hosting providers. Read [this](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#generating-a-new-ssh-key) document to learn how to generate a new SSH key in the GitHub service.
```

Comment:
`(..) an SSH key to authenticate your repository`
-> `(..) an SSH key to authenticate with your repository`

For me without the `with` word the sentence means sth different because you changing the way of authentication
from: **git binary auth in GitHub repository**
to: **GitHub repository auth in git binary**

but maybe I'm wrong, so let's discuss that 🙂

Reply:
oki doki, added `with` 😉